src/WebJobs.Script.WebHost/Controllers/InstanceController.cs

```diff
@@ -79,5 +75,30 @@ public IActionResult GetHttpHealthStatus()
             // Reaching here implies that http health of the container is ok.
             return Ok();
         }
+
+        private async Task<IActionResult> AssignInternal(HostAssignmentContext assignmentContext)
```

Comment:
nit: include `Async` suffix for `Task` returning methods.
```suggestion
        private async Task<IActionResult> AssignInternalAsync(HostAssignmentContext assignmentContext)
```

Reply:
This is removed